src/components/Episode/index.js

```diff
@@ -1,4 +1,6 @@
 import s from "./styles.scss";
+import { TrailerContext, TrailerProvider } from "./context.jsx";
 
 export { s };
+export { TrailerContext, TrailerProvider };
```

Comment:
```suggestion
export { s, TrailerContext, TrailerProvider };
```

Reply:
just found out I could make suggestions like this! looks like I can only make single-line suggestions though...